hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractCreateSuite.java

```diff
@@ -615,6 +618,22 @@ HapiSpec contractWithAutoRenewNeedSignatures() {
                 .then();
     }
 
+    private HapiSpec contractCreateNoncesExternalizationHappyPath() {
+        final var contract = "NoncesExternalization";
+        final var contractCreateFn = "deployContract";
+
+        return defaultHapiSpec("ContractCreateNoncesExternalizationHappyPath")
+                .given(uploadInitCode(contract), contractCreate(contract).via(contractCreateFn))
+                .when()
+                .then(withOpContext((spec, opLog) -> {
+                    HapiGetTxnRecord op = getTxnRecord(contractCreateFn)
+                            .hasPriority(recordWith()
+                                    .contractWithIdHasContractNonces(
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**issue:** ‏Here (i.e. in the nonces map we externalize) we should also have the 3 new contracts with nonce 1.